src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ReviewEvidenceNote.cshtml

```diff
@@ -33,20 +31,46 @@
         @Html.Partial("_ViewEvidenceNotePartial", Model.ViewEvidenceNoteViewModel)
 
         <!-- radiobuttons for approved -->
-        <p>Choose whether to approve, reject or return the evidence note</p>
+        <p class="govuk-!-margin-top-7">Choose whether to approve, reject or return the evidence note</p>
         <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" id="approval-options">
             @Html.Gds().ValidationMessageFor(m => m.SelectedValue)
             @this.WeeeGds().RadioButtonsFor(
                 m => m.SelectedValue,
                 Model.PossibleValues,
-                "Choose whether to approve, reject or return the evidence note",
```

Comment:
Whats the reason this has been removed and replaced with blank string. When the legend text is set and RadioButtonLegend.VisuallyHidden is set an accessible caption is added to the options list.